### DISCUSSION TITLE
GitHub Actions being unreliable in the past few days

### DISCUSSION THREAD

**Discussion by JCMais** [community/community#13360] [Actions] [answered]
Just in the past 2 weeks, there were 6 incidents involving GitHub Actions.

This caused jobs to not start, deployments that rely on GitHub Actions to not run, and PRs approval to be slower, as the job status would not be reported in the UI, thus blocking PRs from being merged.

Are there plans for a post-mortem on those?

Also, #hugops to anyone involved in fixing this, as it seems to be a major issue.

**Comment by JCMais** [comment]
Again, third day in a row: 
![image](https://user-images.githubusercontent.com/1041647/159945682-01db9486-354c-4386-995a-b0f0f21bb2b4.png)


What is going on with GitHub?

**Reply by lukearmstrong** [reply]
![image](https://user-images.githubusercontent.com/306764/160120444-0e67dec2-30d5-474b-84d9-8ee4cd7e2476.png)

Gone down again today, "scheduled maintenance".

We use actions to run tests and deploy to unique test links when we open a PR, then on merge to main we deploy to production, it feels like every day for the past 2 weeks it's has been down when we've needed to use it.

I thought I was doing the right thing automating everything with actions so other developers on the team didn't have to run tests locally or manually deploy things, but I feel like an idiot for ultimately blocking us every time this goes down.

I don't want to, but we are going to have to switch to Jenkins as this is too unreliable.

**Comment by thclark** [comment]
It would really be good to know if we're going to get a post-mortem on this, transparency breeds confidence.

Honestly I wouldn't mind paying a bit more for reliable actions but at the moment this is costing a lot of dev time.

Does anyone have self-hosted actions workers? Have they been unreliable too?

**Comment by ethomson** [comment, marked as answer]
Hi everyone. Thanks for the question. We post regular service updates on [our blog](https://github.blog/), including a post about [our recent service disruptions](https://github.blog/2022-03-23-an-update-on-recent-service-disruptions/) and will share more in our March Availability Report in the next few weeks.

**Reply by JCMais** [reply]
thanks @ethomson!

**Reply by skipjack** [reply]
I'm seeing a similar issue for a few PRs (but not all) today that are all targeted to the same base branch where the workflow is configured. Not seeing anything on the blog or GitHub status page, did I miss it?

**Comment by thclark** [comment]
Thanks @ethomson - hadn't seen that blog. I find it way easier to be sympathetic with when reading things like that - reminds me that GitHub's DB/solution architects are mere mortals just like me. If you should see the state of my event handling queue... ;)
